src/frontend/efiling-demo/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "efiling-demo",
   "version": "0.6.0",
-  "homepage": "/efiling-demo",
+  "homepage": ".",
```

Comment:
question: is the `.` required ? I think it is but cannot remember why...

Reply:
Yes this was the only way it worked

Reply:
Sounds good ;-)